packages/cli/src/commands/search/dump.ts

```diff
@@ -0,0 +1,169 @@
+import {Command, flags} from '@oclif/command';
+import {AuthenticatedClient} from '../../lib/platform/authenticatedClient';
+import {
+  Preconditions,
+  IsAuthenticated,
+} from '../../lib/decorators/preconditions/';
+import {Config} from '../../lib/config/config';
+import PlatformClient from '@coveord/platform-client';
+import {writeFile} from 'fs-extra';
+import {Parser} from 'json2csv';
+import {cli} from 'cli-ux';
+import {
+  buildAnalyticsFailureHook,
+  buildAnalyticsSuccessHook,
+} from '../../hooks/analytics/analytics';
+
+interface SearchResult {
+  raw: {rowid: string};
+}
+
+interface ProgressBar {
```

Comment:
Why do we need a custom interface?

Reply:
It's untyped from oclif (everything set as any). 

It's ultimately coming from another lib (cli-progress) that oclif uses. 

The alternative is to install @types/cli-progress, and tell eslint to remove warnings about extraneous import (we don't rely on cli-progress directly).